app/grants/templates/grants/index.html

```diff
@@ -57,119 +57,175 @@ <h1 class="font-title-xl font-weight-semibold">{% trans 'Provide sustainable fun
 
       <div class="container-fluid py-4 grants-results">
         <div class="container">
-          <strong>Gitcoin Grants</strong> has facilitated the distribution of <strong>${{grant_amount|floatformat:0}}k</strong> to Open Source since it's launch in January 2019.  
-          <a class="font-body" href="/results">See more results</a>.
-          <br>
-          <img src="{% static 'v2/images/grants/robots.png' %}" style="max-width: 100px;" class="mt-1" />
-
-        </div>
-      </div>
-
-      <div class="container-fluid py-5 grants-matching">
-        <div class="container">
-          <div class="row">
-          <div class="col-md-6">
-            <h3 class="mb-1 font-subtitle">Thats a wrap!</h3>
-            <p class="mb-2 mt-2">
-            $100k Round 3 has closed!  From 9/16 through 10/2 we were matching contributions with quadratic funding. We plan to announce the winners soon.
+          <div class="col-12">
+            <p class="font-smaller-1">
+              <strong>Gitcoin Grants</strong> has facilitated the distribution of
+              <strong>${{grant_amount|floatformat:0}}k</strong> to Open Source since
+              it's launch in January 2019.
+              <a href="/results">See more results</a>.
             </p>
+            <img src="{% static 'v2/images/grants/robots.png' %}" style="max-width: 150px;" class="mt-1" />
+          </div>
 
-              Gitcoin Grants is the largest pilot for Quadratic Funding around! Want to help fund open source in the 'mathematically optimal' way?
-              </a>
+          <div class="col-md-12 font-smaller-1 text-center mt-4">
+            <p class="mb-1 font-weight-semibold">
+              $100k Round 3 has closed!
+            </p>
+            <p class="mb-1">
+              From 9/16 through 10/2 we were matching contributions with quadratic
+              funding for which we've announced the winners
+            </p>
+            <p>
+              Gitcoin Grants is the largest pilot for Quadratic Funding around!
+              Want to help fund open source in the 'mathematically optimal' way?
             </p>
 
             {% if not user.is_authenticated %}
-              <a class="btn btn-gc-blue" href="mailto:founders@gitcoin.co">{% trans 'Become a Matching Partner' %}</a>
+              <a class="btn btn-gc-blue font-body font-weight-semibold" href="mailto:founders@gitcoin.co">
```

Comment:
maybe target  _blank  to avoid people losing the page?

Reply:
It's a mailto so that opens in a separate window all-together